pkg/networkpolicy/multinetingressrule.go

```diff
@@ -149,3 +225,27 @@ func (builder *IngressRuleBuilder) GetIngressRuleCfg() (*v1beta1.MultiNetworkPol
 
 	return builder.definition, nil
 }
+
+func (builder *IngressRuleBuilder) validate() (bool, error) {
+	objectName := "MultiNetworkPolicyIngressRule"
```

Comment:
```suggestion
	objectName := "multiNetworkPolicyIngressRule"
```
same as before, this will become the start of an error message so should start with a lowercase letter

Reply:
done